src/relay/backend/contrib/ethosn/codegen.cc

```diff
@@ -713,9 +713,17 @@ runtime::ethosn::OrderedCompiledNetwork EthosnCompiler::CompileEthosnFunc(const
   auto network_with_ids = ConstructNetwork(mod, gvar, func);
   // Now set the required build flags
   sl::CompilationOptions options = CreateOptions();
-  // Finally compile the network
+  // Set the experimental compiler if enabled, for now this is not part of the
```

Comment:
my thinking was to reduce the scope of the environment variable to the support library, since its not prefixed in any way there's always the (small!) chance the same variable can be used elsewhere. We could move the scope to the NPU codegen instead

Reply:
Another option could be to look for this option from just the tvmc frontend functions and set/unset it from python for the entire invocation of the tvmc. Maybe that leaves the EthosN code cleaner. Are there any downsides of doing that though? :thinking: